arrows/vtk/applets/fuse_depth.cxx

```diff
@@ -81,6 +82,10 @@ bool check_config(kv::config_block_sptr config)
     validate_required_input_dir("input_cameras_directory", *config,
                                  main_logger) && config_valid;
 
+  config_valid =
+    validate_required_input_file("input_geo_origin_filename", *config,
```

Comment:
This should probably be `validate_optional_input_file` because the geo origin only required if you are also writing a LAS file.  Actually, if you move this check later after checking the file extension below you could have something like:
```c++
if (extension == ".las" )
{
  config_valid =
    validate_required_input_file("input_geo_origin_filename", *config,
                                 main_logger) && config_valid;
}
else
{
  config_valid =
    validate_optional_input_file("input_geo_origin_filename", *config,
                                 main_logger) && config_valid;
}
```